src/modules/common/transaction.options.ts

```diff
@@ -0,0 +1,15 @@
+import { TokenTransfer } from '@multiversx/sdk-core/out';
```

Comment:
maybe drop `/out` ? I've seen this change in Claudiu's PR

Reply:
done